Model/Source/ProductList.php

```diff
@@ -26,33 +19,36 @@ class ProductList
     const CROSSSELL = 'crosssell';
     const WIDGET = 'widget';
 
+    /**
+     * @return array
+     */
     public function toOptionArray()
     {
-        return array(
-            array(
+        return [
+            [
                 'value' => '',
-                'label' => __('None')
-            ),
-            array(
+                'label' => __('None'),
+            ],
+            [
                 'value' => self::CATEGORY,
-                'label' => __('Category and Search Result Pages')
-            ),
-            array(
+                'label' => __('Category and Search Result Pages'),
+            ],
+            [
                 'value' => self::UPSELL,
-                'label' => __('Upsells on Product Pages')
-            ),
-            array(
+                'label' => __('Upsells on Product Pages'),
+            ],
+            [
                 'value' => self::RELATED,
-                'label' => __('Related Products on Product Pages')
-            ),
-            array(
+                'label' => __('Related Products on Product Pages'),
+            ],
+            [
                 'value' => self::CROSSSELL,
-                'label' => __('Cross Sells on Cart Page')
-            ),
-	        array(
-		        'value' => self::WIDGET,
-		        'label' => __('Product List Widget')
-	        ),
-        );
+                'label' => __('Cross Sells on Cart Page'),
```

Comment:
In Magento, they use the name format of "Cross-Sell Products" (dash)

Reply:
Fixed.